crowdsec-docs/versioned_docs/version-v1.2/local_api/database.md

```diff
@@ -4,11 +4,11 @@ title: Databases
 sidebar_position: 2
 ---
 
-By default, the crowdsec Local API use `SQLite` as backend storage. But in case you expect a lot of traffic on your local API, you should use `MySQL` or `PostgreSQL`.
+By default, the CrowdSec Local API uses `SQLite` as backend storage. But in case you expect a lot of traffic on your local API, you should use `MySQL` or `PostgreSQL`.
 
-For `SQLite`, there is nothing to do to make it work with crowdsec. But for `MySQL` and `PostgreSQL` , you have to create the database and the user.
+For `SQLite`, there is no specific configuration required to make it work with CrowdSec. But for `MySQL` and `PostgreSQL` , you have to create the database and the user.
```

Comment:
I would maybe phrase it simpler: "No specific configuration is required for `SQLite` to work with CrowdSec" ? But I let you choose what "sounds" best for you.